### DISCUSSION TITLE
Builds stuck in queued status

### DISCUSSION THREAD

**Discussion by ted-nz** [codemagic-ci-cd/codemagic-docs#1790] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

Seem to be same issue as https://github.com/orgs/codemagic-ci-cd/discussions/1752

Start one or new builds via UI

### Expected results

Builds to start

### Actual results

All builds stuck in queued status

### Build id (optional)

648f9b242f9c1f10214dfc8c

**Comment by maks** [comment]
@ted-nz Thanks for reporting the issue. There does currently seem to be an issue with builds not running correctly which we're looking into at the moment. Apologies for the inconvenience.

**Reply by wudys** [reply]
I'm also experiencing the same issue.
(build id: 648fa9d8e45fb6d28d6cd341)

**Reply by Okunamutata** [reply]
Same here 
Build id: 648fa6250ad91f54bd631e58

**Reply by bryce-walton** [reply]
I'm also experiencing the same issue.
(build id: 648fb601ba567ae06ad66af0)

**Reply by anhartasman** [reply]
same here
build id : 648f8e8020708459bad6ce4e

**Reply by wonzee** [reply]
Same here. Hopefully it becomes normal soon.

**Reply by Buddybetadmin** [reply]
Experiencing the same issue

**Reply by 202100299** [reply]
same here
build id : 648fb93e625b38d09cc2e7e6

**Reply by AathifMahir** [reply]
Experiencing the same here

**Reply by doiyuki** [reply]
same here
build id: 648fc470cd4357f303d68f0e

**Reply by wonzee** [reply]
Still in investigation? When is the estimated time of recovery? @maks

**Reply by maks** [reply]
@wonzee yes sorry its being worked on at the moment, we'll post here once there are further updates.

**Reply by maks** [reply, marked as answer]
Builds should now be running again. Please note it will take some time for all the queued builds to start.
